.coveragerc

```diff
@@ -3,4 +3,6 @@ branch = True
 include =
     invoke/*
     tests/*
-omit = invoke/vendor/*
+omit =
+    invoke/shims.py
```

Comment:
Coverage is omitted for vendor-ed dependencies because those are separately developed modules. I don't think this should be omitted